Dockerfile fixes

osfameron opened the pull request:
* add -y to allow apt-get install to run unattended
 * chmod wait-for-couchbase.sh (this is done in script, but that doesn't seem to be sufficient)

Still has remaining issue on startup:

	try-cb-api  | standard_init_linux.go:219: exec user process caused: exec format error
	try-cb-api exited with code 1

osfameron committed:
Dockerfile fixes

 * add -y to allow apt-get install to run unattended
 * chmod wait-for-couchbase.sh (this is done in script, but that doesn't seem to be sufficient)

Still has remaining issue on startup:

	try-cb-api  | standard_init_linux.go:219: exec user process caused: exec format error
	try-cb-api exited with code 1

maria-robobug commented:
Just need to add `#!/bin/bash` at the top of `wait-for-couchbase.sh`, should work with that.

osfameron committed:
reinstate shebang link on wait-for-couchbase.sh

This file lost its shebang, leading to error:

	try-cb-api  | standard_init_linux.go:219: exec user process caused: exec format error

Will need re-testing on wsl2.

osfameron commented:
Good spot @maria-robobug ! Pushed additional fix.
@programmatix did you remove that because it caused issues on wsl2?

programmatix commented:
The "-y" thing was a quick fix that I should have tested, apologies.  #!/bin/bash might have got removed during a 3 hour trial-and-error/head-bashing session trying to get past various issues - didn't realise it had caused the "standard_init_linux.go:219: exec user process caused: exec format error" problem.  Thanks for getting this over the last hurdles.

programmatix committed:
Merge branch 'main' into main
